.circleci/images/x86_64-linux-fedora/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM fedora:26
```

Comment:
This needs to be Fedora 27 as the goal is to support ncurses 6.

Alternatively we could use Debian Stretch.